src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
+
+  let sum = 0;
+
+  men.forEach((m) => sum += m.died - m.born);
```

Comment:
Avoid `forEach`. Use `reduce` instead.
For clarifying steps you can firstly create an array with ages (using `map`)
`const menAges = men.map(m => m.died - m.born)`

And then calculate the sum of ages using reduce - https://stackoverflow.com/questions/1230233/how-to-find-the-sum-of-an-array-of-numbers